crossfire/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 from crossfire.clients import AsyncClient, Client  # noqa
 
-NESTED_COLUMNS = ["contextInfo"]
+NESTED_COLUMNS = {"contextInfo", "transports", "victims", "animalVictims"}
```

Comment:
I have added all columns that potentialy has nested values.
As a `set()`, BTW...